tests/functional/encryption/test_encryption_configuration_dashboard.py

```diff
@@ -137,13 +140,7 @@ def test_object_storage_encryption_configuration_dashboard(self, setup_ui_class)
             3. verify encryption data with the nooba and storagecluster spec.
         """
         # Navigate to the Object Storage page
-        object_details_page = (
-            PageNavigator()
-            .nav_storage_cluster_default_page()
-            .nav_storage_systems_tab()
-            .nav_storagecluster_storagesystem_details()
-            .nav_details_object()
-        )
+        object_details_page = DataFoundationTabBar().nav_object_tab()
```

Comment:
was this tested? usually all navigation happens through PageNavigator, but not DataFoundationTabBar. I also not sure if Object tab is accessible, because it was moved to PageNavigator as a separate item.